### PR TITLE
KOGITO-4449 Integrate DMN Validator and JIT dmn

### PR DESCRIPTION
**Pull request by tarilabs**
Ref https://issues.redhat.com/browse/KOGITO-4449

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](https://github.com/kiegroup/kogito-runtimes#contributing-to-kogito)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [x] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

### CODE REVIEW

File: jitexecutor/jitexecutor-dmn/src/main/resources/META-INF/native-image/org.kie.kogito/jitexecutor-dmn/reflect-config.json [hunk 1]
```diff
@@ -1248,6 +1251,51 @@
...
+    },
+    {
+      "name":"com.sun.org.apache.xerces.internal.jaxp.DocumentBuilderFactoryImpl",
+      "allDeclaredConstructors" : true,
+      "allPublicConstructors" : true,
+      "allDeclaredMethods" : true,
+      "allPublicMethods" : true,
+      "allDeclaredFields" : true,
+      "allPublicFields" : true
+    },
```
Comment (danielezonca) [inline jitexecutor/jitexecutor-dmn/src/main/resources/META-INF/native-image/org.kie.kogito/jitexecutor-dmn/reflect-config.json:1289]: Have you tested both with jdk8 and jdk11?

**Review by danielezonca — APPROVED** [review]
Thanks for the PR, approving 👍

**Comment by danielezonca** [comment]
Build failed because of timeout

jenkins retest this please

**Review by r00ta — CHANGES_REQUESTED** [review]
Hi @tarilabs , looks great! 
Could you add a couple of lines to the README to document the endpoint and what the user should expect from the response?

File: jitexecutor/jitexecutor-dmn/src/main/resources/META-INF/native-image/org.kie.kogito/jitexecutor-dmn/reflect-config.json (hunk 1 above)
Comment (tarilabs) [reply jitexecutor/jitexecutor-dmn/src/main/resources/META-INF/native-image/org.kie.kogito/jitexecutor-dmn/reflect-config.json:1289]: I've tested it with `graalvm-ce-java11@20.2.0` which AFAIK is the one we should use. Which other GraalVM you want me to test with ?

Comment (danielezonca) [reply jitexecutor/jitexecutor-dmn/src/main/resources/META-INF/native-image/org.kie.kogito/jitexecutor-dmn/reflect-config.json:1289]: Yes that version should be fine. My main concern was about jdk11 differences in XML but if you already tested with `graalvm-ce-java11@20.2.0` we should be fine 👍

**Comment by tarilabs** [comment]
noting green for posterity [link](https://rhba-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/KIE/job/kogito/job/pullrequest/job/kogito-apps/46/display/redirect)
image

**Comment by tarilabs** [comment]
```
Caused by: org.jenkinsci.plugins.workflow.steps.MissingContextVariableException: Required context class hudson.FilePath is missing

Perhaps you forgot to surround the code with a step that provides this, such as: node
```

much wow 🎱 🤷‍♂️ 

Jenkins retest this

**Comment by tarilabs** [comment]
```

Caused by: hudson.plugins.git.GitException: Command "git config remote.origin.url https://github.com/tarilabs/kogito-apps.git" returned status code 255:

stdout: 

stderr: error: could not lock config file .git/config: No space left on device



	at org.jenkinsci.plugins.gitclient.CliGitAPIImpl.launchCommandIn(CliGitAPIImpl.java:2372)
```

🎱 🎱 🤷‍♂️ 

Jenkins retest this

**Comment by tarilabs** [comment]
```
Caused by: hudson.plugins.git.GitException: Command "git config remote.origin.url https://github.com/tarilabs/kogito-apps.git" returned status code 255:
stdout: 
stderr: error: could not lock config file .git/config: No space left on device
```

🎱 🎱 🎱 🤷‍♂️ 

Jenkins retest this

**Comment by tarilabs** [comment]
Jenkins retest this

**Pull request merged**